### DISCUSSION TITLE
Please help me to collect Video Games!! 🎮

### DISCUSSION THREAD

**Discussion by wowry** [NAIST-SE/AttractingContributors#54] [General]
## <p align="center">Please help me to collect Video Games!! 🎮</p>
<p align="center">
  <a href="https://github.com/wowry/AttractingContributors-VideoGames">https://github.com/wowry/AttractingContributors-VideoGames</a>
</p>
<p align="center">
Pull requests are always welcome! :)
</p>

**Comment by nosfill** [comment]
added!

**Reply by wowry** [reply]
Thanks!!

**Comment by satokibi44** [comment]
added!

**Reply by wowry** [reply]
Thanks!

**Comment by tkmaeda** [comment]
added mine just now!

**Reply by wowry** [reply]
Thanks!

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Reply by wowry** [reply]
Thanks!
I have sent a PR!

**Comment by mailtoy** [comment]
added please help me https://github.com/mailtoy/AttractingContributors-Movies

**Reply by wowry** [reply]
Thanks!
I have sent a PR!

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames

**Reply by wowry** [reply]
Thanks!
I have sent a PR.

**Comment by sergiodl-naist** [comment]
Added Touhou 6
Please visit and try to ask a question or recommend me one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Reply by wowry** [reply]
Thanks!
I have sent a PR.
